pom.xml

```diff
@@ -920,6 +926,16 @@
 					<artifactId>schema-to-pojo-maven-plugin</artifactId>
 					<version>${schema-to-pojo.version}</version>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>javacc-maven-plugin</artifactId>
+					<version>2.6</version>
```

Comment:
We can update?

Reply:
This is the latest version of the maven-plugin, even though there are newer versions of javacc.